docs/reference/setup/bootstrap-checks.asciidoc

```diff
@@ -0,0 +1,144 @@
+[[bootstrap-checks]] == Bootstrap Checks
+
+Collectively, we have a lot of experience with users suffering
+unexpected issues because they have not configured
+<<important-setting,important settings>>. In previous versions of
+Elasticsearch, some of these settings were logged as
```

Comment:
In previous versions of Elasticsearch, *forgetting to set* some of these settings caused Elasticsearch to log warnings.